RatScanner/RatScannerMain.cs

```diff
@@ -209,6 +234,23 @@ internal void SetupRatEye()
 
 	private RatEye.Config GetRatEyeConfig(bool highlighted = true)
 	{
+		// Default to our primary screen, then try to set it to game window if we can.
+		int ScreenWidth = Screen.PrimaryScreen.Bounds.Width;

```

Comment:
We use PrimaryScreen size as a default (if game is closed, etc) and then every time the timer fires, if this is incorrect, we will try to use the `gameRect` size instead.